.travis.yml

```diff
@@ -1,10 +1,11 @@
 language: ruby
+
 cache: bundler
+
 rvm:
+  - 2.3.1
```

Comment:
Well I'm taking this out for now, some rainbow gem is unhappy on these rubies. Will fix later.